convert-to-declarative-plugin/pom.xml

```diff
@@ -64,59 +64,16 @@
       <artifactId>workflow-scm-step</artifactId>
       <version>2.7</version>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>htmlpublisher</artifactId>
-      <version>1.21</version>
-      <!-- non mandatory plugin  -->
-      <optional>true</optional>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>build-timeout</artifactId>
-      <version>1.18</version>
-      <!-- non mandatory plugin <optional>true</optional> -->
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>credentials</artifactId>
-      <version>2.1.18</version>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>display-url-api</artifactId>
       <version>2.0</version>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.main</groupId>
-      <artifactId>maven-plugin</artifactId>
```

Comment:
This was only being used for an extension that was disabled, so I just deleted it and the disabled extensions for now.